universum/lib/utils.py

```diff
@@ -26,14 +31,17 @@
     "make_request"
 ]
 
+ReturnT = TypeVar('ReturnT')
```

Comment:
Why not ReturnType and DecoratorType?

Reply:
no special reason, just the convention I've seen in mypy docs.
Would you prefer Type?

Reply:
ReturnT is ok to me.